Admin/assets/firebase/common.js

```diff
@@ -66,6 +66,34 @@ function getShiftName(shiftNumber){
     return "No Info Available"
 }
 
+/**
+ * Gives current time  
+ * @returns {string} time ,current time in HHMMSS & 24 hours format e.g. 180254
+ */
+ function getCurrentTime() {
+    let d = new Date(); // Current 
+    let hourString = ((d.getHours() <= 9) ? '0' : '') + d.getHours().toString();
+    let minuteString = ((d.getMinutes() <= 9) ? '0' : '') + d.getMinutes().toString();
+    let secondString = ((d.getSeconds() <= 9) ? '0' : '') + d.getSeconds().toString();
+    let time = hourString + minuteString + secondString;
+    return time;
+}
+
+/**
+ * Calculates shift name according to given time
+ * @param {string} time in HHMMSS & 24 hours format e.g. 180510  
+ * @returns {string} Morning, Afternoon, Evening
+ */
+ function getGreeting(time) {
```

Comment:
var d = new Date()
var hour = d.getHours()
if hours < 12, mrng
12-16, afternoon
else evning

Lets use this approach to make the decision